src/components/AlertCard/styles.module.css

```diff
@@ -28,40 +32,60 @@
   margin-right: 0.5rem;
 }
 
+.flexGrow {
+  flex-grow: 2;
+  background-color: white;
```

Comment:
Does the flex grow make sure the alert icon stays where it is and not expand? If so, this is fine and we just remove the `background-color` declaration (as explained above). If not, we might be able to remove this whole class.